gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -97,6 +98,16 @@ public RoutingConfiguration getRouting()
         return routing;
     }
 
+    public void setIncludeClusterHostInResponse(boolean includeClusterHostInResponse)
```

Comment:
Please move this getter and setter to the bottom for consistency with field order.